girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -401,7 +401,7 @@ class AnnotationSchema:
         'properties': {
             'type': {
                 'type': 'string',
-                'enum': ['imageoverlay']
+                'enum': ['tiledimage']
             },
```

Comment:
@naglepuff, @jeffbaumes has convinced me that we should just call these annotations `image` and `pixelmap`.  When we call `geo.util.pixelCoordinateParams`, the only difference there is whether a tile size is passed to the function, and a difference in the url having zxy expansions, so it should be relatively easy to make the viewer agnostic to whether the underlying image is tiled or not.

Reply:
Renamed in 24115d5